pkg/restapi/operation/operations.go

```diff
@@ -342,9 +351,15 @@ func (c *Operation) onboardUser(id string) (*user.Profile, error) {
 	return userProfile, nil
 }
 
-// TODO redirect to the UI: https://github.com/trustbloc/hub-auth/issues/39
-func handleAuthResult(w http.ResponseWriter, r *http.Request, _ *user.Profile) {
-	http.Redirect(w, r, "", http.StatusFound)
+func (c *Operation) handleAuthResult(w http.ResponseWriter, r *http.Request, userProfile *user.Profile) {
+	handle := url.QueryEscape(uuid.New().String())
```

Comment:
Is this handle should be fetched from the UI as described here: https://github.com/trustbloc/hub-auth/issues/39